org.gabmus.gfeeds.json

```diff
@@ -2,7 +2,7 @@
     "app-id": "org.gabmus.gfeeds",
     "command": "gfeeds",
     "runtime": "org.gnome.Platform",
-    "runtime-version": "44",
+    "runtime-version": "46",
```

Comment:
can you use 47?

Reply:
There are some errors related to python that prevent using runtime 47